client/src/main/java/org/threadly/litesockets/client/http/HTTPClient.java

```diff
@@ -38,7 +38,7 @@
 import org.threadly.litesockets.protocols.http.shared.HTTPParsingException;
 import org.threadly.litesockets.protocols.http.shared.HTTPRequestMethod;
 import org.threadly.litesockets.protocols.http.shared.HTTPResponseCode;
-import org.threadly.litesockets.protocols.ws.WebSocketFrameParser.WebSocketFrame;
+import org.threadly.litesockets.protocols.ws.WSFrame;
```

Comment:
I personally prefer the longer (more descriptive) name of `WebSocketFrame`, but do agree it should be moved to a top level class.  I don't feel strongly about this, but thought I would mention it.

Reply:
I think we should either prefix the classes as `WebSocket` or we should change the package from `ws` to `websocket`.  At least if the package was `WebSocket` the acronym would be obvious.  Having `ws.WS` seems less friendly to me.  I could go either way on the `WebSocket` to `WS` prefix, though do slightly prefer the longer class names.

Reply:
change ws => websocket for the package name.